### PR TITLE
Release 0.5.0

### LINKED ISSUE #58: Release 0.5.0
## Depends on tasks:
- #33
- #38 
- #39
- #42 
- #43
- #47

 
## Description:
Prepare the documentation and source for release.

## User requirements:
User should be able to go to the release page in repo, get the information about changes that were done and be able to download/setup the app

## Dev requirements and restrictions:

- Create a new branch for the release with the name "release-0.5.0" and set it up for release;
- Create a new branch to bump the version, if requires.
- Check the CHANGELOG.md, verify that there are no missed changes, and update with release information;
- Add the necessary info to the [Release_notes](https://github.com/IronFoundation/iOS-Woof/blob/main/release_notes.md) in a user-friendly format based on CHANGELOG.md.
- Before merging to the main branch set up the tag "v.0.5.0" on the last commit before the merge commit.
- Publish the Release under the Release [section on GitHub.](https://github.com/IronFoundation/iOS-Woof/releases)

### PR DESCRIPTION
**Pull request by iBYarchitect**
## Summary
This PR is a part of the work to resolve #58 

## Changes
- Updated release_notes.md
- Updated CHANGELOG.md
- Changed build configuration to release.

## Checks to complete
- [x] The PR title has a conventional commit style (short and descriptive).
- [x] The PR description includes essential information that helps understand the purpose of the changes.
- [x] I've built and run my changes, and no warnings or errors occurred.
- [x] All existing tests passed with my changes.
- [x] I've updated `CHANGELOG.md` file.
- [x] PR has assignee, reviewers, milestone, it is properly linked to the project and to the issue.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,13 +6,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
...
 
-- Add new templates for `epic` issue and `feature request` - [#4](https://github.com/IronFoundation/iOS-Woof/pull/4)
-- Add the possibility for sitters to update info about themselves - [#52](https://github.com/IronFoundation/iOS-Woof/pull/52)
-- Add enums `AppButton` and `AppAlert` which contains repeating strings and text in buttons - [#55](https://github.com/IronFoundation/iOS-Woof/pull/55)
-- Add the possibility for owners to filter sitters by city - [#56](https://github.com/IronFoundation/iOS-Woof/pull/56)
-- Add the development server for test goals - [#57](https://github.com/IronFoundation/iOS-Woof/pull/57)
+- Added a city filter to the list of sitters - [56](https://github.com/IronFoundation/iOS-Woof/pull/56)
+- Add new templates for `epic` issue and `feature request` - [4](https://github.com/IronFoundation/iOS-Woof/pull/4)
+- Add the possibility for sitters to update info about themselves - [52](https://github.com/IronFoundation/iOS-Woof/pull/52)
+- Add enums `AppButton` and `AppAlert` which contain repeating strings and text in buttons - [#55](https://github.com/IronFoundation/iOS-Woof/pull/55)
```
Comment (NadzeyaShpakouskaya) [inline CHANGELOG.md:16]: Why did you remove `#` when in the previous releases we use format with `#`
```
- Added a city filter to the list of sitters - [#56](https://github.com/IronFoundation/iOS-Woof/pull/56)
```

File: Woof/Woof.xcodeproj/xcshareddata/xcschemes/Woof.xcscheme [hunk 2]
```diff
@@ -23,7 +23,7 @@
       </BuildActionEntries>
    </BuildAction>
    <TestAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "Release"
```
Comment (NadzeyaShpakouskaya) [inline Woof/Woof.xcodeproj/xcshareddata/xcschemes/Woof.xcscheme:26]: Do we need to switch a build configuration to Release?

**Review by NadzeyaShpakouskaya — COMMENTED** [review]
I have some small questions:

File: CHANGELOG.md (hunk 1 above)
Comment (iBYarchitect) [reply CHANGELOG.md]: It's just a typo.
Fixed with 9361860

File: Woof/Woof.xcodeproj/xcshareddata/xcschemes/Woof.xcscheme (hunk 2 above)
Comment (iBYarchitect) [reply Woof/Woof.xcodeproj/xcshareddata/xcschemes/Woof.xcscheme:26]: Isn't this one of the goals of this PR?

Comment (iBYarchitect) [reply Woof/Woof.xcodeproj/xcshareddata/xcschemes/Woof.xcscheme]: Fixed with 46a2635

**Review by NadzeyaShpakouskaya — APPROVED** [review]
LGTM
